bcreg-aca/config/schemas.yml

```diff
@@ -80,105 +80,105 @@
+#       required: true
+#     civic_address:
+#       label_en: Civic Address
+#       description_en: Street number, name, direction, suite, etc.
+#       required: true
+#     municipality:
+#       label_en: Municipality
+#       description_en: Name of a recognized municipality
+#       required: true
+#     province:
+#       label_en: Province
+#       description_en: Name of recognized sub-national territory
+#       required: true
+#     postal_code:
+#       label_en: Postal Code
+#       description_en: Identifier issued by postal agency
+#       required: true
+#     country:
+#       label_en: Country
+#       description_en: Name of a recognized national territory
+#       required: true
+#     address_effective_date:
+#       label_en: Address Effective Date
+#       description_en: Date current address became/becomes effective
+#       required: false
+#     effective_date:
+#       label_en: Credential Effective Date
+#       description_en: Date Credential is effective
+#       required: true
+#     expiry_date:
+#       label_en: Credential Expiry Date
+#       description_en: Date Credential expires
+#       required: false
+#     reason_description:
+#       label_en: Credential Issue Reason
+#       description_en: Reason for credential issue/update
+#       required: false
 
-- name: relationship.registries.ca
-  version: '1.0.42'
-  label_en: Relationship
-  description_en: Relationship Credential
-  attributes:
-    registration_id:
-      label_en: Registration ID
-      description_en: Unique identifer assigned to entity by registrar
-      required: true
-    associated_registration_id:
-      label_en: Associated Registration ID
-      description_en: Registry id(s) of associated organizations/individuals
-      required: true
-    associated_registration_name:
-      label_en: Associated Registration Name
-      description_en: Registered name of associated organizations/individuals
-      required: false
-    relationship:
-      label_en: Relationship
-      description_en: Name of the relationship
-      required: true
-    relationship_description:
-      label_en: Relationship Description
-      description_en: Description of the relationship
-      required: true
-    relationship_status:
-      label_en: Relationship Status
-      description_en: Status of the relationship
-      required: true
-    relationship_status_effective:
-      label_en: Relationship Status Effective
-      description_en: Date the relationship became/becomes effective
-      required: false
-    effective_date:
-      label_en: Effective Date
-      description_en: Date Credential is effective
-      required: true
-    expiry_date:
-      label_en: Credential Expiry Date
-      description_en: Date Credential expires
-      required: false
-    reason_description:
-      label_en: Reason
-      description_en: Reason for credential issue/update
-      required: false
+# - name: relationship.registries.ca
```

Comment:
We need to keep the `relationship.registries.ca` credential, it is used to define the relationships between Sole props and their parent corp.

Reply:
Ooops! I definitely went comment-happy there - thanks for catching this!